recipes/qtlocation/bld.bat

```diff
@@ -0,0 +1,28 @@
+@echo on
+mkdir build
+if errorlevel 1 exit /B 1
+cd build
+if errorlevel 1 exit /B 1
+
+qmake ..\qtlocation.pro
+if errorlevel 1 exit /B 1
+
+nmake
```

Comment:
Please use `jom -j%CPU_COUNT%` and add jom to requirements

Reply:
Sounds good! Thank you for the review